apps/o2-blocks/src/spoiler/editor.jsx

```diff
@@ -87,7 +87,16 @@ registerBlockType( 'a8c/spoiler', {
 	icon: 'warning',
 	category: 'a8c',
 	description: __( 'Hide content until the reader wants to see it.' ),
-	keywords: [ __( 'spoiler' ), __( 'accordion' ), __( 'Dropdown' ) ],
+	keywords: [
+		__( 'spoiler' ),
+		__( 'accordion' ),
+		__( 'Dropdown' ),
+		__( 'details' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **38** times:
`translate( 'Details:' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/pull/59636)